docs/metrics/doid.md

```diff
@@ -32,11 +32,11 @@
 
 | Metric | Value |
 | ------ | ----- |
-| AnnotationAssertion | 99692 |
-| SubAnnotationPropertyOf | 1 |
+| AnnotationAssertion | 103824 |
```

Comment:
This increase might be accounted for in part by the addition of `OMO_0003012` since there are 3,906 of these annotation assertions in "tmp/component-download-doid.owl-owl". However, in "components/doid.owl" synonyms that are abbreviations, e.g. AIDS for DOID:635 are still missing.

Reply:
> This increase might be accounted for in part by the addition of OMO_0003012 since there are 3,906 of these annotation assertions in "tmp/component-download-doid.owl-owl".

That sounds likely to me!